lab_1_classify_by_unigrams/start.py

```diff
@@ -3,18 +3,46 @@
 """
 # pylint:disable=too-many-locals, unused-argument, unused-variable
 
+from lab_1_classify_by_unigrams.main import (collect_profiles, create_language_profile,
+                                             detect_language, detect_language_advanced,
+                                             print_report, tokenize)
+
 
 def main() -> None:
     """
     Launches an implementation
     """
     with open("assets/texts/en.txt", "r", encoding="utf-8") as file_to_read_en:
         en_text = file_to_read_en.read()
+        profile_en = create_language_profile("en", en_text)
     with open("assets/texts/de.txt", "r", encoding="utf-8") as file_to_read_de:
         de_text = file_to_read_de.read()
+        profile_de = create_language_profile("de", de_text)
     with open("assets/texts/unknown.txt", "r", encoding="utf-8") as file_to_read_unk:
         unknown_text = file_to_read_unk.read()
-    result = None
+        unknown_profile = create_language_profile("unknown", unknown_text)
+    result = en_text
```

Comment:
so, you will come to the defense and say that your result is `en_text` ?

Reply:
yes, i am afraid of none because my mypy discouraged me without return

Reply:
done